docs/development/issues.md

```diff
@@ -1,11 +1,18 @@
 # Issues In Electron
 
+* [Where to Find and Submit Issues](#where-to-find-and-submit-issues)
 * [How to Contribute to Issues](#how-to-contribute-to-issues)
 * [Asking for General Help](#asking-for-general-help)
 * [Submitting a Bug Report](#submitting-a-bug-report)
 * [Triaging a Bug Report](#triaging-a-bug-report)
 * [Resolving a Bug Report](#resolving-a-bug-report)
 
+## Where to Find and Submit Issues
```

Comment:
I don't think there needs to be a new section for this, just add a link to the first instance of:

> new issue in the `electron/electron` issue tracker.

Reply:
PR updated as recommeded.